### PR TITLE
Initialize the page for creating an obproxy cluster and handle some details

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary

1. Encapsulate the `PreText` component.
2. Detach the `namespace` selection box.
3. Initialize the page for creating an `obproxy cluster`.
4. Some `UI` optimizations.

### CODE REVIEW

File: ui/src/components/AlertDrawer/index.tsx [hunk 1]
```diff
@@ -13,7 +13,12 @@ export default function AlertDrawer({
   ...props
 }: AlertRuleDrawerProps) {
   return (
-    <Drawer closable={false} onClose={onClose} {...props}>
+    <Drawer
```
Comment (powerfooI) [inline ui/src/components/AlertDrawer/index.tsx:16]: Add `maskClosable={false}` here to avoid accident closing.

**Review by powerfooI — CHANGES_REQUESTED** [review]

**Review by yezaoshu — DISMISSED** [review]
+1

File: ui/src/components/AlertDrawer/index.tsx (hunk 1 above)
Comment (yang1666204) [reply ui/src/components/AlertDrawer/index.tsx:16]: Done.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**